web_hierarchy/static/src/js/web_hierarchy_view.js

```diff
@@ -0,0 +1,98 @@
+ /*
+ * Copyright 2019 Kevin Kamau <kevkamaa96@gmail.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+ */
+
+ /*eslint no-undef: "google", "drawChart" */
+
+odoo.define('web.HierarchyView', function (require) {
+    'use strict';
+
+    var core = require("web.core");
+    var View = require('web.View');
+    var _lt = core._lt;
+    var qweb = core.qweb;
+
+    var HierarchyView = View.extend({
+        template: 'HierarchyView',
+        icon: "fa-sitemap",
+        searchable: false,
+        mobile_friendly: true,
+        require_fields: true,
+        display_name: _lt('Hierarchy'),
+        render_chart: function (size) {
+            var self = this;
+            var size_val = !size ? "medium": size; /*eslint no-negated-condition: "error"*/
```

Comment:
Here can use short-circuit evaluation
```suggestion
            var size_val = size || "medium";
```

Reply:
done